Add atlas to db feature addition and documentation

bkieft-usa opened the pull request:
Removing previous request due to branch update and being out of sync with main. Adding documentation for running Add_Atlas_to_DB.ipynb so anyone can deposit their favorite csv compound table as an atlas. Also updated the notebook itself to sort the csv before adding it to the database and added checks on parameters/formatting.

tharwood3 commented:
Looks good! The docs are really nice and I appreciate the extra checking step added to the end of the notebook. I had two very minor things I think would be worth changing. We probably should have the sort_atlas_csv function in a .py file, metatlas.io.metatlas_get_data_helper_fun might make sense here -- that way it is easier to track with git. My only other suggestion would be to clear outputs on the notebook before committing.

bkieft-usa commented:
Good suggestions, I moved the sort function to metatlas.io.metatlas_get_data_helper_fun.py. Cleared the notebook outputs and updated the docs markdown. Anything else you can think of?

tharwood3 commented:
Nice. I can't think of anything else, feel free to merge into main whenever you are ready. Thanks!
